opensearch/serializer.py

```diff
@@ -163,6 +163,13 @@ def loads(self, s, mimetype=None):
         if not mimetype:
             deserializer = self.default
         else:
+            # Treat 'application/vnd.[opensearch|elasticsearch]+json'
+            # as application/json for compatibility.
+            if mimetype == "application/vnd.opensearch+json":
```

Comment:
What do you think about:
```
            if mimetype in {"application/vnd.opensearch+json", "application/vnd.elasticsearch+json"}:
                mimetype = "application/json"
```
(also can reuse constant mentioned in the comment ^)